docs/en/guides/webhooks/batch-deliveries.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Webhooks Batch Deliveries
+pagename: webhooks-batch-deliveries
+lang: en
+---
+
+# Batch Delivery
+
+## Enabling Batch Delivery
+
+You can choose to configure your webhook so that it will make requests with a batched payload format, by selecting the "Deliver in Batched Format" option.
+If enabled, at the time each request is formed, it will batch up to 50 pending deliveries into its payload.
+
+This can help with managing the processing of large bursts of deliveries as a result of many SG events being generated in a short amount of time. Under typical system operation, multiple deliveries per payload will occur when subscribed events are generated more frequently than once in a 2 second period.
+
+![Batch Delivery Enabled](./images/batch_delivery_enabled.png)
+
+## Responding to deliveries
+
+If enabling batched deliveries, we recommend that your receiving service design guarantees a response much faster than 1 second per event. Otherwise, you will be at increased risk for timeouts and webhook failure when batches are large.
```

Comment:
```suggestion
If enabling batched deliveries, we recommend that your receiving-service-design guarantees a response much faster than 1 second per event. Otherwise, you will be at increased risk for timeouts and webhook failure when batches are large.
```